static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -1,3 +1,150 @@
 .DevHub-MyAddons-copy {
   max-width: 32em;
 }
+
+.DevHub-MyAddons-item {
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e4e4e4;
+  padding-bottom: 25px;
+}
+
+.DevHub-MyAddons-item-modified {
+  margin-left: auto;
+  align-self: flex-end;
+}
+
+.DevHub-MyAddons-item-details {
+  margin-left: 25px;
+
+  .DevHub-MyAddons-item-name {
+    font-size: 24px;
+    font-weight: 400;
+  }
+
+  a {
```

Comment:
Are pseudo-classes defined elsewhere and/or does this need anything specific for that?